webpack.config.js

```diff
@@ -33,3 +33,7 @@ const modeConfig = production
   : require('./config/webpack.config.dev')
 
 module.exports = merge(withTarget, modeConfig)
+
+if (require.main === module) {
```

Comment:
ici la commande est `node ./webpack.config.js`, ça peut valoir le coup de mettre dans le commentaire du commit si c'est le cas.

Reply:
`feat: Show config if executed via node webpack.config.js ` is not clear enough ?